google/cloud/bigquery/external_config.py

```diff
@@ -897,8 +894,7 @@ def csv_options(self, value):
 
     @property
     def google_sheets_options(self) -> Optional[GoogleSheetsOptions]:
-        """Optional[google.cloud.bigquery.external_config.GoogleSheetsOptions]:
-        Additional properties to set if ``sourceFormat`` is set to
+        """Additional properties to set if ``sourceFormat`` is set to
         GOOGLE_SHEETS.
```

Comment:
(nit) Does this fit into a single line?

Reply:
Not within 79 characters, which is what I've got VS Code configured to wrap at.

Looks like Sphinx did the right thing. Probably because there's no blank line.

![Screen Shot 2021-09-30 at 11 52 47 AM](https://user-images.githubusercontent.com/247555/135498099-2747f526-ecf6-4132-b227-7c1394466c17.png)

Reply:
Now that surprised me, but even better then!